tests/unit_tests/test_game_rating.py

```diff
@@ -984,10 +1047,292 @@ async def test_single_wrong_report_still_rated_correctly(game: Game, player_fact
+        assert result.outcomes == {**team1_outcomes, **team2_outcomes}
+
+
+async def test_rating_adjustment_1v1_newbies(rating_service, player_factory):
+    IGNORED = object()
+
+    # Both players have no ratings yet
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={1: {}, 2: {}},
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                    2: pytest.approx((1235, 429), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={1: pytest.approx((1766, 429), abs=1)},
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_newbies_global_pros(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # Both players have no ladder rating, but high global rating
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {RatingType.GLOBAL: Rating(2000, 75)},
+            2: {RatingType.GLOBAL: Rating(1900, 75)},
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                    2: pytest.approx((1235, 429), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={},
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_pros_global_newbies(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # Both players have high ladder rating but no global rating
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {RatingType.LADDER_1V1: Rating(2000, 75)},
+            2: {RatingType.LADDER_1V1: Rating(1900, 75)},
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((2011, 75), abs=1),
+                    2: pytest.approx((1889, 75), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={
+                    1: pytest.approx((2038, 348), abs=1),
+                    2: pytest.approx((1340, 419), abs=1),
+                },
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_pros_global_mixed(
```

Comment:
I would add another test where everything is the same, but player 2 has lower global rating to show that the global adjustment of player 1 is not dependend on the other's global rating